techniques/failures/F30.html

```diff
@@ -52,92 +49,79 @@
-(<strong>Failure</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/audio-only-and-video-only-prerecorded">1.2.1: 音声のみ及び映像のみ (収録済)</a>
+(<strong>失敗</strong>)
     </li>
   
   </ul>
   
 
 
 
-<p>This failure applies to all technologies.</p></div>
+<p>この失敗例は、全ての技術に適用される。</p></div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p> This describes a failure condition for all techniques involving text
-                        alternatives. If the text in the "text alternative" cannot be used in place
-                        of the non-text content without losing information or function then it fails
-                        because it is not, in fact, an alternative to the non-text content. </p>
-   </section><section id="examples"><h2>Examples</h2>
-      <section class="example" id="example-1"><h3>Example 1</h3>
+<section id="description"><h2>解説</h2>
+      <p>これは、テキストによる代替を使用している全てのテクニックに対する失敗条件について述べている。"テキストによる代替" の中のテキストを、情報又は機能を失うことなく非テキストコンテンツの代わりに使用することができない場合、事実上、非テキストコンテンツに対する代替物としては意味をなさなくなる。</p>
+   </section><section id="examples"><h2>事例</h2>
+      <section class="example" id="example-1"><h3>事例 1</h3>
          
-            <p> Examples of text that are not text alternatives include: </p>
+            <p>テキストによる代替とはならないテキストの事例は次のとおり:</p>
             <ul>
-               <li> placeholder text such as " " or "spacer" or "image" or
-                                        "picture" etc that are put into the 'text alternative'
-                                        location on images or pictures. </li>
-               <li> programming references that do not convey the information or
-                                        function of the non-text content such as "picture 1",
-                                        "picture 2" or "0001", "0002" or "Intro#1", "Intro#2". </li>
-               <li> filenames that are not valid text alternatives in their own
-                                        right such as "Oct.jpg" or "Chart.jpg" or
-                                        "sales\\oct\\top3.jpg" </li>
+               <li>画像や写真の「テキストによる代替」の位置に埋め込まれた、" "、"スペーサー"、"画像"、"写真" といったプレースホルダーテキスト。</li>
+               <li>非テキストコンテンツの情報又は機能を伝達しない、"picture 1" 及び "picture 2" 、"0001" 及び "0002"、あるいは "Intro#1" 及び "Intro#2" といったプログラム上の参照情報。</li>
+               <li>"Oct.jpg"、"Chart.jpg"、あるいは "sales\\oct\\top3.jpg" といった、それ自体では有効なテキストによる代替とならないファイル名</li>
             </ul>
          
       </section>
    </section>
 
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li> Check each text alternative to see if it is not actually a text
-                                    alternative for the non-text content. </li>
+            <li>テキストによる代替それぞれについて、実際にはその非テキストコンテンツに対するテキストによる代替ではないことを確認する。</li>
```

Comment:
check の訳を、他のテクニックと合わせました。
```suggestion
            <li>テキストによる代替それぞれについて、実際にはその非テキストコンテンツに対するテキストによる代替ではないことをチェックする。</li>
```